bertytech/content/newsletter/news-60/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Berty News #60"
+date: 2021-06-11
+aliases: ["/newsletter/weekly-60"]
+---
+
+# News #60
+
+Hi cool kids 🏴☠️
+
+Here's a newsletter that will make you happy: Berty is on TestFlight and will be soon on the Play Store.🤩🤩🤩🤩. It's coming soon. We know we announced in our previous newsletter that It was supposed to arrive last week. But there is a little delay in the validation and some bugs to fix. Anyway, if you have an iPhone, you can download the alpha! (Read below how!)
+
+Just a quick word about the team before letting you dive into this newsletter: everyone is doing well. We admit that the restaurants and bars are finally open again until 11pm. It feels so good to be back to normal. We can't wait to celebrate the release of our Alpha community. We are very excited to showcase our work. 😊
+
+
+## 🚀 Open Source
+
+### Berty Alpha Community
+    Alpha is ONLY available on iOS (TestFlight) for now.
+    Android version is coming soon.
+    Join our Discord to be notified. 
+
+**Why a community alpha version?**
+
+We decided an app version made to open doors to developers and testers, and to contribute on the development. Please note that the app will host data/metadata to help developers to go forward during the Alpha version. It’s only a TEST version to allow community to have testing sessions, explore code, UX/UI, and improve all kinds of bugs.
+
+Our goal with the Alpha Community: 
+* Every community members can download the Alpha
+* This version is not ready for daily usage, it’s just a Alpha version to discover the Berty app.
+* TOR feature is not ready yet. IPs are not hidden.
+* Account Persistence : not ready yet, your data will be deleted for next updates
```

Comment:
`your data will be deleted for next updates` -> `your data will probably be lost during updates`